artifacts/testdata/server/testcases/ntfs2.in.yaml

```diff
@@ -0,0 +1,43 @@
+Parameters:
```

Comment:
this was renamed to ntfs_ea.in.yaml

is this a new file or just a change from the old one?

Reply:
no we can remove it.